### DISCUSSION TITLE
Font Server Tokens Studio Sync Issue

### DISCUSSION THREAD

**Discussion by vtutkaBayer** [six7/figma-tokens#1490] [Q&A]
We're running into an issue getting Tokens Studio to pull fonts from our font server, Suitcase Fusion. It will pull fonts locally and it will pull Google fonts from Figma (I'm assuming), but our font server fonts don't seem to be accessible through the Tokens Studio plugin. 

Example: 
 },
  "Display 1": {
    "value": {
      "fontFamily": "Helvetica Neue LT Pro",
      "fontWeight": "300",
      "lineHeight": "116",
      "fontSize": "96",
      "letterSpacing": "-1.25"
    },
    "type": "typography"
  },
  
Helvetica Neue Lt Pro is loaded into Suitcase Fusion (the font server), and while I can use it in Figma, I cannot use it via the Tokens Studio Plugin
![Screen Shot 2022-12-20 at 3 10 00 PM](https://user-images.githubusercontent.com/112118021/208767106-54555064-3f29-422e-a6e6-518725ff3034.png)

**Comment by stephen-salyer** [comment]
Would love an answer for Extensis and/or Font Servers in general.

**Reply by josephstephens87** [reply]
Seconded! This would be great to know.

**Reply by stephen-salyer** [reply]
@six7 should we make a feature request for font server integration? or does that exist already?

**Reply by six7** [reply]
This is not something that we as plugin authors can control. All we do is tell Figma 'Apply that font with that weight'.

In the screenshot above: Have you tried setting it to `exactly the font family / weight` as Figma allows you to pick from? I believe 300 is not a weight that is available in Figma itself, could that be it?

If you could provide a screenshot of what Figma allows you to choose from for that I could help debug, but a font server integration is not something plugin authors would need to do. As long as that font is supported from Figma, it should work.
